lib/chat_api/uploads.ex

```diff
@@ -0,0 +1,60 @@
+defmodule ChatApi.Uploads do
+  @moduledoc """
+  The Uploads context.
+  """
+
+  import Ecto.Query, warn: false
```

Comment:
I'm not sure we're using Queries in this file yet, since we are only dealing with Repo and Schema changeset, guess we can avoid importing it here for now...